.github/workflows/CI_Pipeline.yml

```diff
@@ -25,20 +25,19 @@ jobs:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag my-test-image .
     - run: docker run --env-file .env.test  my-test-image tests
-    - run: docker run -e TRELLO_KEY=${{secrets.TRELLO_KEY}}  -e TRELLO_TOKEN=${{secrets.TRELLO_TOKEN}} -e TRELLO_BOARD_ID=${{secrets.TRELLO_BOARD_ID}} my-test-image tests_e2e
-
-  deploy:
-    name: Deploy build package to Heroku
-    needs: build
-    if: ${{ github.event_name ==  'push' && github.ref == 'refs/heads/exercise-8' }}
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2 # Adds a step to checkout the repository code
+    - run: docker run -e MONGO_DB_NAME=${{secrets.MONGO_DB_NAME}} -e MONGO_CONNECT=${{secrets.MONGO_CONNECT}} -e MONGO_COLLECTION_NAME=${{secrets.MONGO_COLLECTION_NAME}} -e MONGO_DEFAULT_STATUS=${{secrets.MONGO_DEFAULT_STATUS}} my-test-image tests_e2e
+#  deploy:
+#    name: Deploy build package to Heroku
+#    needs: build
+#    if: ${{ github.event_name ==  'push' }}
+#    runs-on: ubuntu-latest
+#    steps:
+#    - uses: actions/checkout@v2 # Adds a step to checkout the repository code
```

Comment:
Why is this commented out?

Reply:
Hi Hugh, 

Thanks for the feedback, I had popped a message on slack regarding this and it seems this GitHub action no longer works and when I was searching it seemed to be related to the recent security issue so I went down the route of commenting out for the time being.